lib/hyrax/transactions/steps/save_collection_banner.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+module Hyrax
+  module Transactions
+    module Steps
+      ##
+      # Adds banner info via `ChangeSet`.
+      #
+      # During the update collection process this step is called to update the file
+      # to be used as a the banner for the collection.
+      #
+      class SaveCollectionBanner
```

Comment:
This needs tests.  You might be able to copy in the tests from the controller and make some minor adjustments for them to work for the transaction code.